.github/workflows/lint.yml

```diff
@@ -0,0 +1,17 @@
+name: Lint
+on:
+  push:
+    tags:
+      - v*
```

Comment:
what is the motivation for running on tag?  tag does not change any of the repo content, so I'm not sure why we would need to test again

Reply:
I can remove it. I followed the same pattern we had on `karina`

Reply:
from a build job?  We need to build on tag

Reply:
Ah okay I see